sponsorship.md

```diff
@@ -10,12 +10,24 @@ As a free-to-attend gathering, our attendees don't have to worry about huge conf
 
 In exchange for financial backing, we would provide:
 
+**in the basic sponsorship package:**
+
 - a mention at the beginning and end of the conference
-- your logo on our website
+- your logo on conference website
 - mentions on social media channels
-- your name in emails about SoCraTes UK, reaching a significant audience including the [London Software Craftsmanship Community][London Software Craftsmanship Community] mailing list with over 3200 craftsmen and women
+- your name in emails about SoCraTes UK, reaching a significant audience including the [London Software Craftsmanship Community][London Software Craftsmanship Community] mailing list with over 3500 craftsmen and women
 - your merchandise available to everyone at the conference
 
+**in the extended sponsorship package:**
+
+- All options from the basic package
+- Secured places for 2 participants from your company. Note you will still need to purchase tickets in addition to the sponsorship.
+
+**in the premium sponsorship package:**
+
+- All options from the basic package
+- Secured places for 4 participants from your company. Note you will still need to purchase tickets in addition to the sponsorship.
```

Comment:
would include a `:` here and above: 
`Note: you will still need to purchase tickets in addition to the sponsorship.`